EmpireLegends/Player.cpp

```diff
@@ -234,3 +384,77 @@ Resources* Player::getResources() const
 	return pResources;
 }
 
+int Player::getNumofcards() {
+	return NumCards;
+}
+
+void Player::setNumofcards() {
+	NumCards++;
+}
```

Comment:
I feel like we can just use the card verctor's length for `getNumofcards`, so that we can drop `setNumofcards` and `NumCards`.

*sidenote:* `setNumofcards` isn't really a the right name for incrementing an integer I think